### ISSUE TITLE
What's the best practice if I want to make styling modifications to the UI library?

### ISSUE THREAD

**Issue by samleeiii** [magento/devdocs#613]
in vendor/magento/magento2-base/lib/web/css/source i want to overwrite some of those less files, what's the best way so i don't get hosed everytime i do a software update? :)

**Comment by samleeiii** [comment]
Realized the files are actually in lib/web/css/source/lib/. I ended up commenting stuff out and putting what I need in my theme less files, but yeah what's the best practices?

**Comment by almarchenko** [comment]
hi @samleeiii 
The best practices are described here 
http://devdocs.magento.com/guides/v2.0/frontend-dev-guide/css-guide/css_quick_guide_approach.html
Please let me know if it helps.

**Comment by samleeiii** [comment]
So I'm familiar with what's outlined in the doc, but it only allows me to extend or overwrite a parent theme. It doesn't let me overwrite what's in the lib/web/css/source... But from my experimenting it seems like there's some dependency on the variables in these less files and it doesn't compile right if they're not present, so maybe that's why we're not able to overwrite those?

**Comment by almarchenko** [comment]
@samleeiii I think this paragraph talks about overriding library items:
http://devdocs.magento.com/guides/v2.0/frontend-dev-guide/css-guide/css_quick_guide_approach.html#structured_changes

**Comment by samleeiii** [comment]
I remember trying that but let me try again and get back to you.
